starter-code/index.html

```diff
@@ -21,14 +22,17 @@
         <h2>The shocking power of static pages</h2>
       </section>
       <nav class="main-nav">
+        <div class="icon-menu">
+          
+        </div>
```

Comment:
Well done! Since the content is defined for us in our icon fonts, the line space here is not necessary.